### PR TITLE
Create frequently-asked-questions-about-fsrs.md

### PR DESCRIPTION
**Pull request by Expertium**
I want to transfer this section of the GitHub guide here: https://github.com/open-spaced-repetition/fsrs4anki/blob/main/docs/tutorial.md#faq

It has a lot of questions, so I did my best to select the ones I consider to be especially important and/or frequently asked.

### CODE REVIEW

**Comment by Expertium** [comment]
@brishtibheja @L-M-Sherlock feedback is welcome

File: src/frequently-asked-questions-about-fsrs.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Frequently asked questions about FSRS
+
+***
+
+Q1: Which platforms support FSRS?
```
Comment (brishtibheja) [inline src/frequently-asked-questions-about-fsrs.md:5]: Is this really a FAQ? I don't think people ever inquire this.

File: src/frequently-asked-questions-about-fsrs.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+A1: FSRS is supported on all platforms: desktop Anki (Windows, Mac, and Linux), AnkiWeb (browser), AnkiMobile (iOS), and AnkiDroid (Android). Make sure that your Anki(Mobile) version is 23.10 or newer, and your AnkiDroid version is 2.17 or newer.
+
+***
+
+Q2: My first interval for 'Good' is 3-5 days, and my first interval for 'Easy' is over a week long! Is this normal?
```
Comment (brishtibheja) [inline src/frequently-asked-questions-about-fsrs.md:11]: either bold (when thought of as UI elements) or double quotes. single quotes aren't used that much.

```suggestion
Q2: My first interval for **Good** is 3-5 days, and my first interval for **Easy** is over a week long! Is this normal?
```

File: src/frequently-asked-questions-about-fsrs.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+A2: Yes. FSRS tends to give longer first intervals than SM-2, but for mature cards the opposite is true - FSRS is more conservative.
+
+If you wish to make the intervals shorter, you can increase your desired retention. But note that this will shorten all the intervals, not just the first intervals.
+
+***
+
+Q3: I only use 'Again' and 'Good', will FSRS work fine?
```
Comment (brishtibheja) [inline src/frequently-asked-questions-about-fsrs.md:19]:
```suggestion
Q3: I only use **Again** and **Good**, will FSRS work fine?
```

File: src/frequently-asked-questions-about-fsrs.md (hunk 1 above)
Comment (Expertium) [reply src/frequently-asked-questions-about-fsrs.md:5]: https://www.reddit.com/r/Anki/comments/1b89wt1/how_come_i_still_cant_download_the_fsrs_ankidroid/
https://www.reddit.com/r/Anki/comments/1b7x9ni/fsrs_on_different_versions_of_anki/
https://www.reddit.com/r/Anki/comments/1aocrce/is_it_advisable_to_keep_fsrs_on_since_ankidroid/
https://www.reddit.com/r/Anki/comments/1anl69o/is_fsrs_now_integrated_into_anki/
https://www.reddit.com/r/Anki/comments/1akzd2e/fsrs_and_ankidroid/
https://www.reddit.com/r/Anki/comments/1akphto/fsrs_on_ios/
https://www.reddit.com/r/Anki/comments/1aje203/ankidroid_update/
https://www.reddit.com/r/Anki/comments/1blvmhs/does_the_ankipc_settings_sync_with_ios_anki_and/
https://www.reddit.com/r/Anki/comments/1bkm88o/potential_issues_with_fsrs/
https://www.reddit.com/r/Anki/comments/1bufpkv/how_does_anki_keep_fsrs_updated/
https://www.reddit.com/r/Anki/comments/1bsgel6/is_it_possible_to_use_multiple_versions_of_anki/
https://www.reddit.com/r/Anki/comments/1bp1tli/how_can_i_anki_on_multiple_devices/
https://www.reddit.com/r/Anki/comments/1cph43b/switching_between_anki_versions_on_same_computer/
https://www.reddit.com/r/Anki/comments/1byy5c7/using_ankidroid_mess_up_fsrs/
https://www.reddit.com/r/Anki/comments/1e7h9ud/fsrs_vs_ankidroid/
https://www.reddit.com/r/medicalschoolanki/comments/1d21vyv/ankidroid_along_with_pc_anki_with_fsrs/
https://www.reddit.com/r/Anki/comments/19a259u/does_fsrs_sync_properly_between_ios_ankimobile/

I believe this answers your question (wink)

Comment (brishtibheja) [reply src/frequently-asked-questions-about-fsrs.md:5]: are these still being asked after a year?

Comment (Expertium) [reply src/frequently-asked-questions-about-fsrs.md:5]: Most of these are 5-8 months old. It's true that questions about FSRS support on different paltforms are asked less frequently these days, but I still feel like it's worth mentioning. Let's see what @dae thinks

File: src/frequently-asked-questions-about-fsrs.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Q6: Does FSRS take into account the amount of time that I spend reviewing a card?
+
+A6: No, FSRS only uses interval lengths and grades. However, the amount of time you spend on reviews is used when calculating optimal retention using the "Compute minimum recommended retention" feature.
+
+***
+
+Q7: Do I need 1000 reviews before I can optimize parameters?
+
+A7: That was the case in earlier versions of Anki. In the most recent version, the optimizer can be used with any number of reviews.
```
Comment (L-M-Sherlock) [inline src/frequently-asked-questions-about-fsrs.md:45]: The threshold is removed in Anki 24.06.3.

File: src/frequently-asked-questions-about-fsrs.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+A2: Yes. FSRS tends to give longer first intervals than SM-2, but for mature cards the opposite is true - FSRS is more conservative.
+
+If you wish to make the intervals shorter, you can increase your desired retention. But note that this will shorten all the intervals, not just the first intervals.
+
+***
+
+Q3: I only use **Again** and **Good**, will FSRS work fine?
+
+A3: Yes. In some cases, FSRS may even be more accurate if you only use 2 buttons.
```
Comment (L-M-Sherlock) [inline src/frequently-asked-questions-about-fsrs.md:21]: `2 buttons` is ambiguous. Using `hard` and `easy` is also `2 buttons`.

File: src/frequently-asked-questions-about-fsrs.md (hunk 4 above)
Comment (Expertium) [reply src/frequently-asked-questions-about-fsrs.md:45]: Do you want me to specify the exact version 
?

Comment (brishtibheja) [reply src/frequently-asked-questions-about-fsrs.md:45]: Do it. There are a lot of people on previous versions. Apparently cuz of add ons.

**Comment by Expertium** [comment]
@dae this PR has been (almost) ready for a few days now, would be nice if you answered one last question - should we keep the question-answer pair about availability of FSRS on different platforms - and merged it.
Also this https://github.com/ankitects/anki-manual/pull/264

**Comment by dae** [comment]
> should we keep the question-answer pair about availability of FSRS on different platforms

I'm sorry, I'm not sure what you're asking me there. The PR otherwise looks ok.

**Comment by Expertium** [comment]
@dae I meant this
> Q1: Which platforms support FSRS?

> A1: FSRS is supported on all platforms: desktop Anki (Windows, Mac, and Linux), AnkiWeb (browser), AnkiMobile (iOS), and AnkiDroid (Android). Make sure that your Anki(Mobile) version is 23.10 or newer, and your AnkiDroid version is 2.17 or newer.

**Comment by dae** [comment]
By the way, I recall talk of content in the FAQs page vs the manual, but not sure where I saw it. Recently, I've come to feel that a FAQ page is more a convenience for support people, and the info is better placed in the manual instead. In the future, I'd like to see us improving the manual, and removing the entries of the FAQ page.

**Comment by Expertium** [comment]
I don't understand why the manual and FAQ have different Github repositories in the first place

**Pull request merged**